<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeMapTextureBrowser</class>
 <widget class="QDialog" name="CubeMapTextureBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>732</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cube Map Texture Browser</string>
  </property>
  <widget class="QLineEdit" name="textRootPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSelectRootPath">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>8</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReload">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>8</y>
     <width>116</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCreateCube">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>8</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>New Cubemap</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listTextures">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1000</width>
     <height>683</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
